mysql-test/suite/rocksdb/t/ddse_upgrade_row_lock.test

```diff
@@ -0,0 +1,36 @@
+--source include/have_rocksdb.inc
+--source include/have_innodb_ddse.inc
+
+--let $MYSQLD_DATADIR= `select @@datadir`
+
+# backup data
+let $tmp_dir = $MYSQLTEST_VARDIR/tmp;
+let $saved_datadir = $tmp_dir/saved_datadir;
+force-cpdir $MYSQLD_DATADIR $saved_datadir;
+
+# There are 3 places during DD upgrade which may involve a lot row locks
+# - insert select statement: copy data from one storage engine to another
+#       storage engine, this only happens for DD SE upgrade. with this change,
+#       no row lock is hold
+# - drop DD tables: when drop old DD table, SQL layer read one row and delete it
+#       This happens for both DD SE upgrade and DD version upgrade. max row lock
+#       required is the max number of rows among these DD tables
+# - SYSTEM table update: there are a lot sql scripts involves insert data into
+#       system table. This happens for both DD SE upgrade and DD version upgrade
+#       Currently some tables may involves 15-20K rows locks during insert data
```

Comment:
```suggestion
#       Currently some tables involve 15-20K rows locks during data insertion
```

Reply:
Since I replied to an "obsolete" comment, it may not be visible, I'll repeat here: I think it's still better to shutdown first, then copy, or we may end up inadvertently testing it as a "hot backup" by cp, which is not always consistent. It's just the shutdown does not need a specific timeout value.